application/src/main/java/org/togetherjava/tjbot/commands/Colors.java

```diff
@@ -0,0 +1,81 @@
+    public static final Color COLOR_FLAMINGO = new Color(0xf0c6c6);
+    public static final Color COLOR_PINK = new Color(0xf5bde6);
+    public static final Color COLOR_MAUVE = new Color(0xc6a0f6);
+    public static final Color COLOR_RED = new Color(0xed8796);
+    public static final Color COLOR_MAROON = new Color(0xee99a0);
+    public static final Color COLOR_PEACH = new Color(0xf5a97f);
+    public static final Color COLOR_YELLOW = new Color(0xeed49f);
+    public static final Color COLOR_GREEN = new Color(0xa6da95);
+    public static final Color COLOR_TEAL = new Color(0x8bd5ca);
+    public static final Color COLOR_SKY = new Color(0x91d7e3);
+    public static final Color COLOR_SAPPHIRE = new Color(0x7dc4e4);
+    public static final Color COLOR_BLUE = new Color(0x8aadf4);
+    public static final Color COLOR_LAVENDER = new Color(0xb7bdf8);
+    public static final Color COLOR_TEXT = new Color(0xcad3f5);
+    public static final Color COLOR_SUBTEXT1 = new Color(0xb8c0e0);
+    public static final Color COLOR_SUBTEXT0 = new Color(0xa5adcb);
+    public static final Color COLOR_OVERLAY2 = new Color(0x939ab7);
+    public static final Color COLOR_OVERLAY1 = new Color(0x8087a2);
+    public static final Color COLOR_OVERLAY0 = new Color(0x6e738d);
+    public static final Color COLOR_SURFACE2 = new Color(0x5b6078);
+    public static final Color COLOR_SURFACE1 = new Color(0x494d64);
+    public static final Color COLOR_SURFACE0 = new Color(0x363a4f);
+    public static final Color COLOR_BASE = new Color(0x24273a);
+    public static final Color COLOR_MANTLE = new Color(0x1e2030);
+    public static final Color COLOR_CRUST = new Color(0x181926);
+
+    // General colors
+    public static final Color SUCCESS = COLOR_GREEN;
+    public static final Color WARNING = COLOR_PEACH;
+    public static final Color FAILURE = COLOR_MAROON;
+
+    // Feature colors
+    public static final Color MOD_AUDIT_LOG = COLOR_BLUE;
+
+    public static final Color REMIND = COLOR_YELLOW;
+
+    public static final Color WOLFRAM_ALPHA = COLOR_SKY;
+
+    public static final Color TEX_BACKGROUND = COLOR_BASE;
+    public static final Color TEX_FOREGROUND = COLOR_TEXT;
+
+    public static final Color SCAM_BLOCKER = COLOR_LAVENDER;
+
+    public static final Color MODERATION = COLOR_MAUVE;
+
+    public static final Color TAG = COLOR_MAROON;
+
+    public static final Color MEDIA_ONLY = COLOR_PEACH;
+
+    public static final Color BLACKLISTED_ATTACHMENT = COLOR_PEACH;
+
+    public static final Color MODMAIL = COLOR_MANTLE;
+
+    public static final Color LOG_TRACE = COLOR_SKY;
+    public static final Color LOG_DEBUG = COLOR_SAPPHIRE;
+    public static final Color LOG_INFO = SUCCESS;
+    public static final Color LOG_WARN = WARNING;
+    public static final Color LOG_ERROR = FAILURE;
+    public static final Color LOG_FATAL = COLOR_RED;
+
+    public static final Color ROLE_SELECT = COLOR_TEAL;
+
+    public static final Color HELP = COLOR_YELLOW;
```

Comment:
this is not how it was supposed to be, each feature can use a color no need to make a new variable in a util file for each feature

im very against this

Reply:
This way you can easily change the color palette in the future

Reply:
y would u want to do that?? make a var at top of each feature for the color it is going to use
this is supposed to be a file which contains color pallet and doesn't get updated unless we decide to switch to a new theme

@Zabuzard whats ur opinion on this??

Reply:
Yea this is too many colours, adding a colour pallette helps against the 5000 different colours, we should limit it to 5 or something

(a kind of) green for success
(a kind of) red for failure
etc

Reply:
@Tais993 vote for one dark then

Reply:
If you're talking about the suggestion you made in the Discord, read my message again, I'm quite saying the opposite of that. I want colours, but not 500 colours, instead like 5

Reply:
one dark have 7 is that ok?

Reply:
Didn't know you were talking about the issue, I gave my vote there.